<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>437</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lable_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>624</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>490</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>20</horstretch>
      <verstretch>22</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ConstVersion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Версия :</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_MaxPower">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>259</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>MAX мощность =</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_Version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>624</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_APP">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>▲</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_MinPower">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>255</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>MIN мощность = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ValueProgressBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>490</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_STOP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>611</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ConstName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>126</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Название : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DOWN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>▼</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lable_ValueMaxPower">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>45</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lable_ValueMinPower">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>45</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting_Pult">
    <property name="title">
     <string>V1.0</string>
    </property>
   </widget>
   <addaction name="menuSetting_Pult"/>
  </widget>
  <action name="actionV1_0">
   <property name="text">
    <string>V1.0</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
